measures/2018/measures-data.json

```diff
@@ -28336,7 +28336,7 @@
     "measureId": "251",
     "description": "This is a measure based on whether quantitative evaluation of Human Epidermal Growth Factor Receptor 2 Testing (HER2) by immunohistochemistry (IHC) uses the system recommended in the current ASCO/CAP Guidelines for Human Epidermal Growth Factor Receptor 2 Testing in breast cancer",
     "nationalQualityStrategyDomain": "Effective Clinical Care",
-    "measureType": "structure",
+    "measureType": "process",
```

Comment:
is this wanted?

Reply:
good q, cc @sarahnw

Reply:
251 is now a process measure according to the latest file on the resource library.